extensions/Worker.Extensions.Rpc/src/ConfigurationExtensions.cs

```diff
@@ -6,7 +6,10 @@
 
 namespace Microsoft.Azure.Functions.Worker.Extensions.Rpc
 {
-    internal static class ConfigurationExtensions
+    /// <summary>
+    /// contains extension methods for IConfiguration.
+    /// </summary>
+    public static class ConfigurationExtensions
```

Comment:
I'd rather we keep this internal. It isn't the worst thing to duplicate in `DotNetWorker.Grpc`. We can look into better ways to make it DRY later.